Tests/DependencyInjection/FixtureMonologExtensionTest.php

```diff
@@ -162,34 +162,34 @@ public function testServerLog()
 
         $container = $this->getContainer('server_log');
 
-        $this->assertEquals(array(
+        $this->assertEquals([
             '0:9911',
             100,
             true,
-        ), $container->getDefinition('monolog.handler.server_log')->getArguments());
+        ], $container->getDefinition('monolog.handler.server_log')->getArguments());
     }
 
     public function testMultipleEmailRecipients()
     {
         $container = $this->getContainer('multiple_email_recipients');
 
-        $this->assertEquals(array(
+        $this->assertEquals([
             new Reference('mailer'),
             'error@example.com',
-            array('dev1@example.com', 'dev2@example.com'),
+            ['dev1@example.com', 'dev2@example.com'],
             'An Error Occurred!',
             null
-        ), $container->getDefinition('monolog.handler.swift.mail_message_factory')->getArguments());
+        ], $container->getDefinition('monolog.handler.swift.mail_message_factory')->getArguments());
     }
 
     public function testChannelParametersResolved()
     {
         $container = $this->getContainer('parameterized_handlers');
 
         $this->assertEquals(
-            array(
-                'monolog.handler.custom' => array('type' => 'inclusive', 'elements' => array('some_channel')),
-            ),
+            [
```

Comment:
@lyrixx should this be changed? It's not the same as the rest of the style?

Reply:
Let's keep the exact same style :)